lib/plugin.c

```diff
@@ -440,7 +440,7 @@ grn_plugin_open(grn_ctx *ctx, const char *filename)
   } else {
     const char *label;
     label = grn_dl_open_error_label();
-    SERR("%s", label);
+    SERR("%s: <%.*s>", label, filename_size, filename);
```

Comment:
`(int)filename_size`.

This breaks Mroonga CI: https://travis-ci.org/mroonga/mroonga/jobs/538586519#L1680

Reply:
BTW, is this safe?
Does `filename`'s encoding always equal to `grn_ctx->encoding`?
For example, if this is local encoding, we need to convert encoding by `grn_encoding_convert_from_locale()`.

Reply:
Oh, I didn't noticed before that commit. :disappointed: